engines/director/lingo/lingo.cpp

```diff
@@ -947,6 +954,8 @@ Common::String Datum::asString(bool printonly) const {
 
 		s += ")";
 		break;
+	case MENUREF:
+		break;
```

Comment:
This is not quite good and needs to be addressed after the merge. We use this for printing the stack contents